### ISSUE TITLE
FreeMode is True after reset() Function call, but it does not work.

### ISSUE THREAD

**Issue by BenPearson95** [grzegorz103/ngx-chess-board#148]
Hi Grzegorz,

I'm running into a issue where I am trying to clear the Markers (arrows, circles...) off the board, but still keep the board in 'FreeMode', and all pieces in their places.

My logic was too grab the current FEN of the board, then reset everything and then set the FEN to what is was before the reset(), this would only remove the markers. 'FreeMode' is turned off during this Reset(), but it doesn't seem to register it turning off in my component logic. and my attempts to turn it back on within the clearMarkers() function doesn't work either.

I did try wrap the `this.freeMode=true;` inside a quick timeout, but that did not work either...

Would you be able to assist? 

```
@ViewChild('board', {static: false}) board: NgxChessBoardView | undefined;
@Input() freeMode: boolean = true;

clearMarkers() {
    const currentFen = this.board?.getFEN();
    this.board?.reset();
    this.board?.setFEN(currentFen);
    // try to turn 'FreeMode' back on.
    this.freeMode = true;
  }
```

Thanks again! :)

**Comment by grzegorz103** [comment]
Hello Ben.

I copied your code and I noticed freeMode input is resetted in library in 56 line in ```engine-facade.ts```, I don't remember the reason I put it there.
After I commented it I achieved something like you want - the free mode was still on after reseting, and could be turned off just like before resetting. I need to test if removing this line won't break anything. Maybe two way binding would fit in this case with this input. Let's keep this issue open until we figure it out.

Thanks
